bundles/org.openhab.binding.guntamatic/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -421,6 +547,18 @@
 			</options>
 		</state>
 	</channel-type>
+	<channel-type id="controlProgramWOManu" advanced="false">
+		<item-type>String</item-type>
+		<label>Set Program</label>
+		<description>Set Program of the Guntamatic Heating System (OFF, NORMAL, WARMWATER)</description>
+		<state>
+			<options>
+				<option value="0">OFF</option>
+				<option value="1">NORMAL</option>
+				<option value="2">WARMWATER</option>
+			</options>
+		</state>
+	</channel-type>
```

Comment:
Just a remark, not a request for change. The only difference between controlProgram and controlProgramWOManu is th elist of options. A good approach in this case is only one channel type but with dynamically provided state options that your code would build depending on your thing type..